lib/spack/docs/index.rst

```diff
@@ -65,6 +65,7 @@ or refer to the full manual below.
    repositories
    command_index
    package_list
+   gpg
```

Comment:
Or some kind of developer documentation. I actually don't know what it's for so I'm not sure where it best belongs.